test/parallel/test-repl.js

```diff
@@ -428,25 +424,6 @@ const errorTests = [
     send: '\'the \\\n   fourth\' +  \'\t\t\\\n  eye  \'',
     expect: '... ... \'the    fourth\\t\\t  eye  \''
   },
-  // using REPL commands within a string literal should still work
-  {
-    send: '\'\\\n.break',
-    expect: '... ' + prompt_unix
-  },
-  // Using REPL command "help" within a string literal should still work
-  {
-    send: '\'thefourth\\\n.help\neye\'',
-    expect: [
-      /\.break/,
-      /\.clear/,
-      /\.editor/,
-      /\.exit/,
-      /\.help/,
-      /\.load/,
-      /\.save/,
-      /'thefourtheye'/
-    ]
-  },
```

Comment:
This seems to be the only place where we call `.help`. @lance would you be so kind and just change the input so we keep the test?